catalog-item.html

```diff
@@ -10,31 +16,40 @@
 
 <body class="page-body">
     <header class="page-header">
-        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
-            <span class="logo-text">BINABOX</span></a>
         <nav class="top-nav">
+            <a class="main-logo"><img src="Images\logo\binaboxlogo36x40.svg" alt="binabox logo" width="188px" height="42px"></a>

```

Comment:
We don't use units of measurement (like px) in the width and height attributes. The correct way is to just specify the numeric values, e.g., width="188" height="42".